### PR TITLE
entity tracking listener

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/server/world/EntityTrackingAudience.mapping [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_5629 net/minecraft/server/world/EntityTrackingAudience
```
Comment (i509VCB) [inline mappings/net/minecraft/server/world/EntityTrackingAudience.mapping]: So the mappings are going on an *adventure*?

Comment (liach) [reply mappings/net/minecraft/server/world/EntityTrackingAudience.mapping]: What? Don't understand

**Review by haykam821 — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Review by jaskarth — CHANGES_REQUESTED** [review]
Why "audience" instead of "listener"? I'm not sure if I fully understand the meaning here.

**Comment by liach** [comment]
Good point, i can rename to listener. Wait for my free time

**Comment by i509VCB** [comment]
How I see it, an audience would refer to an object that can receive messages in very generic terms.

**Review by YanisBft — APPROVED** [review]

**Review by jaskarth — APPROVED** [review]

**Comment by liach** [comment]
@modmuss50 mind review?

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**